src/PowerShellEditorServices/Services/DebugAdapter/Debugging/BreakpointApiUtils.cs

```diff
@@ -0,0 +1,398 @@
+                    firstArgument: null,
+                    setLineBreakpointMethod);
+            });
+
+            s_setCommandBreakpointLazy = new Lazy<Func<Debugger, string, ScriptBlock, string, int?, CommandBreakpoint>>(() =>
+            {
+                MethodInfo setCommandBreakpointMethod = typeof(Debugger).GetMethod("SetCommandBreakpoint", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, string, ScriptBlock, string, int?, CommandBreakpoint>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, string, ScriptBlock, string, int?, CommandBreakpoint>),
+                    firstArgument: null,
+                    setCommandBreakpointMethod);
+            });
+
+            s_getBreakpointsLazy = new Lazy<Func<Debugger, int?, List<Breakpoint>>>(() =>
+            {
+                MethodInfo removeBreakpointMethod = typeof(Debugger).GetMethod("GetBreakpoints", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, int?, List<Breakpoint>>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, int?, List<Breakpoint>>),
+                    firstArgument: null,
+                    removeBreakpointMethod);
+            });
+
+            s_setBreakpointsLazy = new Lazy<Action<Debugger, IEnumerable<Breakpoint>, int?>>(() =>
+            {
+                MethodInfo removeBreakpointMethod = typeof(Debugger).GetMethod("SetBreakpoints", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Action<Debugger, IEnumerable<Breakpoint>, int?>)Action.CreateDelegate(
+                    typeof(Action<Debugger, IEnumerable<Breakpoint>, int?>),
+                    firstArgument: null,
+                    removeBreakpointMethod);
+            });
+
+            s_removeBreakpointLazy = new Lazy<Func<Debugger, Breakpoint, int?, bool>>(() =>
+            {
+                MethodInfo removeBreakpointMethod = typeof(Debugger).GetMethod("RemoveBreakpoint", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, Breakpoint, int?, bool>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, Breakpoint, int?, bool>),
+                    firstArgument: null,
+                    removeBreakpointMethod);
+            });
+        }
+
+        #endregion
+
+        #region Public Static Properties
+
+        private static Func<Debugger, string, int, int, ScriptBlock, int?, LineBreakpoint> SetLineBreakpointDelegate => s_setLineBreakpointLazy.Value;
+
+        private static Func<Debugger, string, ScriptBlock, string, int?, CommandBreakpoint> SetCommandBreakpointDelegate => s_setCommandBreakpointLazy.Value;
+
+        private static Func<Debugger, int?, List<Breakpoint>> GetBreakpointsDelegate => s_getBreakpointsLazy.Value;
+
+        private static Action<Debugger, IEnumerable<Breakpoint>, int?> SetBreakpointsDelegate => s_setBreakpointsLazy.Value;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'SetBreakpointsDelegate'.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4598462)